custom/abt/messaging/custom_recipients.py

```diff
@@ -39,3 +40,30 @@ def abt_case_owner_location_parent_new_framework(case_schedule_instance):
 
     # Get that location's parent location
     return owner_location.parent
+
+
+def abt_location_case_owner_parent_location_old_framework(handler, reminder):
```

Comment:
I was not sure if support for the old framework is technically needed, but included it anyway.